Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/AsyncPDFRenderer.h

```diff
@@ -84,13 +85,30 @@ struct TileRenderData {
 
 WTF::TextStream& operator<<(WTF::TextStream&, const TileRenderData&);
 
-struct PagePreviewRequest {
+struct PDFPagePreviewRenderKey {
+    PDFDocumentLayout::PageIndex pageIndex { std::numeric_limits<WebKit::PDFDocumentLayout::PageIndex>::max() };
```

Comment:
Since we're in `namespace WebKit` already
```suggestion
    PDFDocumentLayout::PageIndex pageIndex { std::numeric_limits<PDFDocumentLayout::PageIndex>::max() };
```